<launch>
  <arg name="stage" default="2"/>
  <arg name="load_model" default="0"/>
  <arg name="load_episode" default="0"/>
  <arg name="train" default="0"/>
  <arg name="run_type" default="2"/>
  <arg name="plan_modulo" default="20"/>
  <param name="stage_number"  value="$(arg stage)"/>
  <param name="load_model"  value="$(arg load_model)"/>
  <param name="load_episode"  value="$(arg load_episode)"/>
  <param name="train"  value="$(arg train)"/>
  <param name="run_type"  value="$(arg run_type)"/>
  <param name="plan_modulo"  value="$(arg plan_modulo)"/>
  <node pkg="rl_nav" type="turtlebot3_dqn.py" name="turtlebot3_dqn" output="screen" />
</launch>
